CONTRIBUTING.md

```diff
@@ -78,30 +78,31 @@ On a certain cadence, we push all commits from master branch into the live branc
 
 ### How to PR
 1. Fork form the original repository, http://github.com/Samsung/tizen-docs.
+   (Ref. https://help.github.com/articles/fork-a-repo/)
 
-2. Clone the forked repository.
-   ```
-   $ git clone <forked repository URL>
+2. Type `git clone`, and then paste the URL you copied in 1. It will look like this, with your GitHub username instead of `YOUR-USERNAME`:
+   ```bash
+   $ git clone https://github.com/YOUR-USERNAME/tizen-docs
```

Comment:
It looks like the URL is in form https://github.com/YOUR-USERNAME/tizen-docs.git
Don't we need to add `.git`. at the end of the url?

Reply:
You don't need to .git at the end of repo.  
xxx.git is general for a git repo but git recognize a repo without .git.

If you want to change it, I will do it.